test/app/controllers/albums.js

```diff
@@ -0,0 +1,83 @@
+const chai = require('chai'),
+  server = require('./../../../app'),
+  dictum = require('dictum.js'),
+  should = chai.should(),
+  bcrypt = require('bcryptjs'),
+  sessionsManager = require('./../../../app/services/sessionsManager'),
+  errors = require('./../../../app/errors'),
+  simple = require('simple-mock'),
+  fetch = require('node-fetch'),
+  User = require('./../../../app/models').user;
+
+const saltRounds = 10;
+simple.mock(fetch('https://jsonplaceholder.typicode.com/albums'), 'example', { album: 'album title' });
```

Comment:
In this cases I really prefer to mock the actual http request with nock. Otherwise you are coupling the test with the package that we use, if someone changes the package this test will fail.